client/src/setupProxy.js

```diff
@@ -1,9 +1,11 @@
 const { createProxyMiddleware } = require("http-proxy-middleware");
 
-console.log("Setting up a Proxy to localhost:5000");
+const serverPort = process.env.SERVER_PORT;
```

Comment:
Doesn't this need to be 
```suggestion
const serverPort = process.env.SERVER_PORT || 5000;
```

Reply:
You're right! Thank you for catching this.